components/Tabs/src/MDCTabBar.h

```diff
@@ -152,3 +172,28 @@ IB_DESIGNABLE
 - (void)tabBar:(nonnull MDCTabBar *)tabBar didSelectItem:(nonnull UITabBarItem *)item;
 
 @end
+
+/**
+ Fixed configuration options for tab bars.
+
+ The tab bar copies the configuration passed at initialization, so all configuration changes must
+ be made before creating the tab bar. 
+*/
+@interface MDCTabBarConfiguration : NSObject <NSCopying>
```

Comment:
We can pick the most common case for the default.  I would guess Top.

Could we add alignment to MDCTabBarPositioning / MDCTabBarPositioningDelegate? (Which would inherit from UIBarPositioning and UIBarPositioningDelegate respectively)

This matches Apple's use of default, toolbars are at the top and tabbars are at the bottom.